tests/testthat/test-rng.R

```diff
@@ -158,6 +158,26 @@ test_that("Binomial random numbers prevent bad inputs", {
 })
 
 
+test_that("provide nice feedback on integer overflow", {
```

Comment:
Think this test is wrongly named!

Reply:
aha, thanks. Originally I was going to throw a nice error but then worked out I could just get rid of  the problem. I'll rename that...